tests/PKSim.Tests/Core/ExtendedPropertyMapperSpecs.cs

```diff
@@ -144,4 +179,12 @@ protected override void CreateSnapshot()
          _snapshot = new ExtendedProperty {Description = "Description", ReadOnly = true, FullName = "Full Name", ListOfValues = new List<object> {4.5, 6.5}, Name = "Name", Value = 5.5};
       }
    }
+
+   public class When_mapping_snapshot_to_extended_property_string_that_can_be_parsed_as_double : When_mapping_snapshot_to_extended_property<string>
+   {
+      protected override void CreateSnapshot()
+      {
+         _snapshot = new ExtendedProperty {Description = "Description", ReadOnly = true, FullName = "Full Name", Name = "Name", Value = "7E3", Type = ExtendedPropertyType.String};
```

Comment:
This failed before the fix. This is good now
Note that old project will still be wrong so in the case of this issue, the type needs to be added by hand

Reply:
and we cannot assume it's string al the time because we are exporting also extended properties for PI who have numbers if I am not mistaken. Not sure if this is used or not so we could default to string in this case. Let me know